src/apiManager.js

```diff
@@ -0,0 +1,45 @@
+import axios from 'axios'
+
+var opv = 'http://opv_master'
```

Comment:
Would be better to use 'const' to ensure this variable can't be modified and even better to save this information in a configuration file.
Like this https://stackoverflow.com/a/47597586